van-tracker.ino

```diff
@@ -1030,10 +998,10 @@ bool handleLocReq(char* smsSender) {
   char latitude[12];
   char longitude[12];
   char speed[4];
-  char dir[4];   // starts out between 0..359 degrees, i.e. "227" and ends up as direction, i.e. "NW"
+  char dir[4];
 
   if (getGPSLatLonSpeedDir(latitude, longitude, speed, dir)) {
-    strcpy_P(message, PSTR("google.com/search?q="));
+    strcpy_P(message, PSTR("maps.google.com/?ll="));
```

Comment:
wow, interesting.  This will bring up apple maps, or google maps if the native app is installed?  Does this work across any version?  Just tried on Android.  It brings up google maps native app, but on my crappy phone it's slower, and it doesn't drop a pin exactly where it is, like it does with the "/search?q" style link.

Very interesting.  I will try to make this a setting that people can choose.  Problem is space is getting tight on the tiny Nano!  Great find!